src/js/components/Tip/Tip.js

```diff
@@ -43,12 +43,20 @@ const Tip = forwardRef(({ children, content, dropProps, plain }, tipRef) => {
     Children.only(children);
 
   const clonedChild = cloneElement(child, {
-    onMouseEnter: () => setOver(true),
-    onMouseLeave: () => setOver(false),
+    onMouseEnter: () => {
+      if (child.props?.onMouseEnter) child.props.onMouseEnter();
```

Comment:
I think we should swap these two lines. We've generally used the pattern of updating our internal state before calling out.

Reply:
Updated in latest commit